api/nodes/views.py

```diff
@@ -1269,3 +1269,127 @@ def get_queryset(self):
             if addon.config.has_hgrid_files
             and addon.complete
         ]
+
+
+class NodeLogList(generics.ListAPIView, NodeMixin, ODMFilterMixin):
```

Comment:
I think you have to use a list filter for this, since you have outsourced the querying.